lib/middleware/verifySession.js

```diff
@@ -18,7 +18,7 @@ function getSessionCookie(req, res) {
 }
 
 function* getUserId(req, res, next) {
-    const loginReferrerUrl = encodeURI(req.protocol + '://' + req.get('host') + req.originalUrl);
+    const loginReferrerUrl = encodeURI(config.get('BASE_PATH_URL') + req.originalUrl);
```

Comment:
The `BASE_PATH_URL` will be something like `http://kmt.ft.com/` (or at least this is how I use it in the app) and the `req.originalUrl` would look something like this `/myft/:licenceId`, which means that when you put those two together we would have double `/` (`http://kmt.ft.com//myft/:licenceId`)
Can we strip the one of the slashes (either from the `BASE_PATH_URL` or the `req.originalUrl`? I'd suggest doing a check to see if one is ending in `/` and the other one starts with it and then do the stripping.